snippets/RGBToHSV.md

```diff
@@ -0,0 +1,44 @@
+---
+title: RGBToHSV
```

Comment:
Should be `RGBtoHSB` - Hue, Saturation, Brightness is more common and refers to the same thing. Update the file name to match.